modules/Internals/Tools/rfcdiff-1.41-CUSTOM.sh

```diff
@@ -316,7 +316,8 @@ function header(file1, file2) {
 "  <meta http-equiv=\"Content-Style-Type\" content=\"text/css\" /> \n" \
 "  <title>Diff: %s - %s</title> \n" \
 "  <style type=\"text/css\"> \n" \
-"    body    { font-size:16px; margin: 0.4ex; margin-right: auto; } \n" \
```

Comment:
Too small font. No need to patch this script. It will be replaced by a fast Perl script soon.

Reply:
Will revert